src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,20 @@
-export const Tabs = () => {};
+export const Tabs = ({ tab, select, set }) => {
+  const check = tab.title === select;
```

Comment:
Consider comparing the `id` property instead of `title`. Comparing objects or their properties like `title` could lead to unexpected bugs. Use unique identifiers like `id` for comparison. For example, `const check = tab.id === select.id;`